code/datums/ai_law_sets.dm

```diff
@@ -25,7 +25,7 @@
 
 /******************** Nanotrasen/Malf ********************/
 /datum/ai_laws/nanotrasen
-	name = "NT Default"
+	name = "NT Recommended"
```

Comment:
StationGuard does sound good...the AI is basically command's loyal watchdog under NT Default.